syntax/std_fmt_test.go

```diff
@@ -0,0 +1,24 @@
+package syntax
+
+import (
+	"testing"
+)
+
+func TestFmtPrettyDict(t *testing.T) {
+	t.Parallel()
+
+	AssertCodesEvalToSameValue(t, `""`, `//fmt.pretty({'a':1,'b':2,'c':3})`)
+	AssertCodesEvalToSameValue(t, `""`, `//fmt.pretty({'a':1,'b':2,'c':{'d':11,'e':22}})`)
+}
+
+func TestFmtPrettyTuple(t *testing.T) {
+	t.Parallel()
+	AssertCodesEvalToSameValue(t, `""`, `//fmt.pretty((a:1,b:2,c:3))`)
+	AssertCodesEvalToSameValue(t, `""`, `//fmt.pretty((a:1,b:2,c:(d:11,e:22)))`)
+}
+
+func TestFmtPrettyMixed(t *testing.T) {
+	t.Parallel()
+	AssertCodesEvalToSameValue(t, `""`, `//fmt.pretty({'a':1,'b':2,'c':(d:11, e:22)})`)
+	AssertCodesEvalToSameValue(t, `""`, `//fmt.pretty((a:1, b:2, c:{'d':11, 'e':22}))`)
```

Comment:
This doesn't look right; `pretty` shouldn't eval to `""`?

Reply:
The behavior of `pretty` is printing the content. If it returns/evals to a value, it will duplicate the content which looks like:
```
(
        c: 3,
        a: 1,
        b: 2
)
(a: 1, b: 2, c: 3)
```
which is not good. So it returns a `""` which can avoid this issue.

Reply:
Mm, okay, but I don't think it's very helpful to test that a void function doesn't return anything.

Can you capture the string as it's output and assert that it matches the pretty format?

Reply:
Yes, the formatted string has been tested in format_test.go and this test case test 3 data types, `dict`, `tuple` and `set` with calling of `//fmt.pretty`.